explainer.md

```diff
@@ -331,24 +331,53 @@ customElements.define("custom-tablist", CustomTabList);
+| `focus`               | _all focusable elements_                                          |                       | `focus`                            |
+| `select`              | Elements whose computed role supports `aria-selected`             |                       | `click`                            |
+| `dismiss` or `escape` | _all elements_                                                    |                       | `Escape` KeyboardEvent             |
+| `contextMenu`         | _all elements_                                                    |                       | `contextmenu`                      |
+| `increment`           | Elements w/ computed role `progressbar`, `scrollbar`, or `slider` | vertical              | `Up` KeyboardEvent                 |
+|                       | ""                                                                | horizontal LTR        | `Right` KeyboardEvent              |
+|                       | ""                                                                | horizontal RTL        | `Left` KeyboardEvent               |
+|                       | Elements w/ computed role `spinbutton`                            | orientation n/a       | `Up` KeyboardEvent                 |
+| `decrement`           | Elements w/ computed role `progressbar`, `scrollbar`, or `slider` | vertical              | `Down` KeyboardEvent               |
+|                       | ""                                                                | horizontal LTR        | `Left` KeyboardEvent               |
+|                       | ""                                                                | horizontal RTL        | `Right` KeyboardEvent              |
+|                       | Elements w/ computed role `spinbutton`                            | orientation n/a       | `Down` KeyboardEvent               |
+| `scrollByPage`        | TBD (possibly custom scroll views)                                |                       | TBD (possibly `PageUp`/`PageDown`) |
+| `scrollIntoView`      | TBD                                                               |                       | No equivalent DOM event            |
+| `setValue`            | n/a                                                               |                       | No equivalent DOM event            |
+
+Notes on the previous table:
+- KeyboardEvent sequences include keyup/keydown/keypress.
+- Control orientation is determined by the computed value of `aria-orientation` which 
+  defaults to `horizontal` for `progressbar` and `slider`, and defaults to `vertical` for 
+  `scrollbar`.
+- Natural language direction is determined by the computed value of `dir` which defaults 
+  to `ltr`, but can be set to `rtl` for languages like Arabic and Hebrew.
+- The event target for KeyboardEvent sequences is the currently focused element.
+- The event target for `click` and `contextmenu` events is lowest leaf node DOM element in 
```

Comment:
I would have assumed these events would come through to a specific "AX Object" - any reason not to fire them on that object?

Reply:
Perhaps I should say "DOM event target" and "currently focused DOM element." I don't think mentioning AX Object is relevant since it's an implementation detail, right? It may originate on the AX Object in WebKit and Chromium, but not Gecko IIRC. 

In either case, the target that a web author can see is always going to be a DOM element, and the event is always going to be a DOM event. Perhaps I'm misunderstanding your question...?

Reply:
Right - my scare quotes around "AX Object" were trying to suggest that I'm not referring specifically to an `AXObject`, but to a node in the platform's accessibility tree, which is mapped back to a DOM node. The AT's point of regard is, as far as I know, unknown to the browser except for when it fires this type of event. So are we meant to take the event target, translate it to a DOM node, and then check that it's the lowest leaf node DOM element..?

Reply:
Yes. The AX action would happen on the UA-internal AX Object, which almost always has a rendered DOM node associated with it. That DOM element should be the target of the DOM event. For something like a link with a span or two in it, if the AT cursor is focused on the link itself, it could trigger the click or contextmenu event on either the link or one of the inner spans... But lets say the AT user is focused on a single word or even character in a sub-span (via Windows SR arrow key nav or VO's 'interact with' functionality), the click would have to start at the span b/c that's where the mainstream mouse event would start when clicking with a physical mouse.

Reply:
Makes sense - again, perhaps we should be specific about how we go from an event on a node in the accessibility tree to an event in the DOM?

Reply:
Even though this is non-normative, I think that's getting into the weeds with implementation details. WebKit and Chromium might be similar, since they both go through the render object, but Gecko would be totally different.